cmd/blockhashes.cpp

```diff
@@ -60,7 +60,7 @@ int main(int argc, char* argv[]) {
     auto blockhashes_table{txn->open(db::table::kHeaderNumbers)};
     
     try {
-        auto current_block_number{db::stages::get_stage_progress(*txn, db::stages::kBlockHashesKey)};
+        auto current_block_number{db::stages::get_stage_progress(*txn, db::stages::kBlockHashesKey)-1};
```

Comment:
Why are you repeating the execution on the last already processed block ?
If last block processed for stage is 10 then you need to begin with 11 not with 9

Reply:
oops, i inverted it